frontend/src/components/explorer/gemBrowser/MapsAvailable.vue

```diff
@@ -10,7 +10,7 @@
       <table class="table maps-table">
         <tbody class="has-text-left">
           <tr v-for="component in mapsAvailable" :key="component.id">
-            <td> {{ component.customName }} </td>
+            <td style="word-break: break-word;"> {{ component.customName }} </td>
```

Comment:
The code base contains two occurrences of 
> word-wrap: break-word;

It this a chance to create a class (if there is no Bulma equivalent)?

Reply:
This is the inbuilt syntax as described in https://developer.mozilla.org/en-US/docs/Web/CSS/word-break. However, I think it would be better to add a hyphen (maybe refer to https://css-tricks.com/almanac/properties/h/hyphenate/) after the break point. For example, in this screenshot <img width="315" alt="Screenshot 2021-09-22 at 08 46 51" src="https://user-images.githubusercontent.com/5833722/134297465-4edf622f-29cb-433f-a776-3512c20437b9.png">, it might be confusing for the word `Glycosylphosphatidylinositol` without adding a hyphen. For English literature, it is usually expressed as 
```
Glycosylphosph-
atidylinositol
```
when breaking of the word is inevitable. 

Another thing is when there is a single letter in another line after word breaking, e.g. 

<img width="327" alt="Screenshot 2021-09-22 at 08 47 48" src="https://user-images.githubusercontent.com/5833722/134298507-80940250-06b5-4430-9e2c-4b698bbb8506.png">
I'm not sure if there is any easy solution to avoid such cases. If not, I guess adding hyphen rule would be enough.

Reply:
`word-break` and `word-wrap` are two different CSS properties with different effects. But I could create a class for `word-break: break-word;` in this PR if you like and create a separate issue for the `word-wrap: break-word;` cases

Reply:
Even if the CSS rules are different, the aim is the same: fit long words in tight spaces.
It might be that two solutions are needed, which would explain the use of different rules.
It might also be that we come up with a more generalized rule.

Reply:
Sorry @nanjiangshu didn't see your comment before! Commit 0cf2139b7f27ac5b734017e34b220c10b08c1590 should fix the single letter issue

Regarding the hyphen, I thinks it's a good idea! I've tried it out but I can't make it work in a good way while not having the map names overflowing. We could perhaps insert `&hyphen; or &shy; ` in long map names and use `hyphens: manual`. But I feel like that might be over engineering things?

Reply:
Commit bdada9bae56c81e9801c543c38a13a4a6bd47400 includes a generalized rule for word-break. Please check it out and see if you think it looks good

Reply:
> But I feel like that might be over engineering things?

I feel the same, especially that we would then need to be careful when using these hyphenated string in eg queries.

<br>
Commit bdada9b is neat!

Reply:
Hi @inghylt  It seems single letter issue still remains but I think we can ignore it since there is no easy solution.
For the hyphenation, I agree that we should not over engineering by insert &hyphen. However, I've tested with the following css 

```css
.break-word {
  word-break: break-word;
  -webkit-hyphens: auto;
  -moz-hyphens: auto;
  -ms-hyphens: auto;
  hyphens: auto;
}
``` 

And the hyphenation works at least for the web-browser `Safari`.  I would suggest we add it. Also note that ` word-break: break-word;` may not work on IE for windows users according to https://developer.mozilla.org/en-US/docs/Web/CSS/word-break.